app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { CONTACTS } from "./constants";
+import DeleteAlert from "./DeleteAlert";
+import Menubar from "./Menubar";
+import Table from "./Table";
+
+const Contacts = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [showMenubar, setShowMenuBar] = useState(true);
```

Comment:
There is a disparity between the `state` and `setter `name. Please fix it. 

Setter should always be in the `set{PascalCaseOfStateName}` format.